proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/chat/session/SessionCommandHandler.java

```diff
@@ -102,6 +108,9 @@ public void handlePlayerCommandInternal(SessionPlayerCommand packet) {
                 .toServer();
           }
         }
+        if (player.getProtocolVersion().getProtocol() == ProtocolVersion.MINECRAFT_1_20_2.getProtocol()) {
```

Comment:
Be optimistic that the packet is there in 1.20.3 too.
```suggestion
        if (player.getProtocolVersion().compareTo(ProtocolVersion.MINECRAFT_1_20_2) >= 0) {
```